src/sexp/string.rs

```diff
@@ -180,21 +183,19 @@ impl OwnedStringSexp {
 }
 
 unsafe fn str_to_charsxp(v: &str) -> crate::error::Result<SEXP> {
-    unsafe {
-        // We might be able to put `R_NaString` directly without using
-        // <&str>::na(), but probably this is an inevitable cost of
-        // providing <&str>::na().
-        if v.is_na() {
-            Ok(savvy_ffi::R_NaString)
-        } else {
-            crate::unwind_protect(|| {
-                Rf_mkCharLenCE(
-                    v.as_ptr() as *const c_char,
-                    v.len() as i32,
-                    cetype_t_CE_UTF8,
-                )
-            })
-        }
```

Comment:
This is intentional in order to be ready for edition 2024. See their blog post

https://blog.rust-lang.org/inside-rust/2024/03/22/2024-edition-update.html
https://doc.rust-lang.org/nightly/edition-guide/rust-2024/unsafe-op-in-unsafe-fn.html